docs/src/references/mqtt-api.md

```diff
@@ -46,11 +46,11 @@ graph LR
 
 Where the groups are described as follows:
 
-|Group|Description|
-|----|----|
-|root|Base topic to group the identifier and channel under one common namespace|
-|identifier|A descriptor which represents which device/service the channel data is related to|
-|channel|Represents the information, such as telemetry data and commands, related to the **identifier**. Each channel type defines its own sub topic structure and corresponding payload format.|
+| Group      | Description                                                                                                                                                                             |
+|------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| root       | Base topic to group the identifier and channel under one common namespace                                                                                                               |
+| identifier | A descriptor which represents which device/service the channel data is related to                                                                                                       |
+| channel    | Represents the information, such as telemetry data and commands, related to the **identifier**. Each channel type defines its own sub topic structure and corresponding payload format. |
```

Comment:
As with all formatting issues, we will have two camps here :-(.

I don't have a strong preference, but I would avoid to have each writer taking the initiative to rewrite part of the doc for its own eye satisfaction.

Reply:
I would vote for not doing the column width alignment in the markdown as it just clutters git diffs potentially obfuscating other changes. Most modern IDEs will have a markdown render which allows users to view the markdown table as intended, or better displaying it using the tedge-docs repo.

Reply:
Then we have to be careful for auto-formatting by IDEs. I've experienced that VSCode formatted the tables to the way exactly this PR changed.

Reply:
There isn't a default formatter for markdown files in VSCode so most likely it comes from a user-installed formatter (possibly prettier?).